### ISSUE TITLE
Data list and data structure

### ISSUE THREAD

**Issue by opababy** [qijiezhao/s3d.pytorch#7]
Sorry, can anyone give some examples about data lists and data structures, thanks!

**Comment by opababy** [comment]
~/data/UCF-101/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01/i 0 163
~/data/UCF-101/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c02/i 0 122 
~/data/UCF-101/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03/i 0 258 

My list is as above:  RGB_FRAME_PATH LABEL FRAME_NUMBER
But the training loss and accuracy are always zero. Is my structure wrong?

**Comment by tobeatraceur** [comment]
> ~/data/UCF-101/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01/i 0 163
> ~/data/UCF-101/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c02/i 0 122
> ~/data/UCF-101/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03/i 0 258
> 
> My list is as above: RGB_FRAME_PATH LABEL FRAME_NUMBER
> But the training loss and accuracy are always zero. Is my structure wrong?

I also met with this problem, did you solve it?
